packages/ui/v2/modules/booker/DatePicker.tsx

```diff
@@ -130,22 +131,25 @@ const DatePicker = ({
   ...passThroughProps
 }: DatePickerProps & Partial<React.ComponentProps<typeof Days>>) => {
   const browsingDate = passThroughProps.browsingDate || dayjs().startOf("month");
+  const { i18n } = useLocale();
 
   const changeMonth = (newMonth: number) => {
     if (onMonthChange) {
       onMonthChange(browsingDate.add(newMonth, "month"));
     }
   };
 
+  const month = browsingDate
```

Comment:
Is there a possibility where `browsingDate` can be undefined? Looking at the old implementation on line 146 it didn't seem that way? Mainly wondering that since you added the conditional here :)

Reply:
there was this condition so i thought maybe it takes some time to get browsingData
```
 {browsingDate ? (
            <>
              <strong className="text-bookingdarker text-base font-semibold dark:text-white">{month}</strong>{" "}
              <span className="text-bookinglight text-sm font-medium">{browsingDate.format("YYYY")}</span>
            </>
          ) : (
            <SkeletonText className="h-8 w-24" />
          )}
```

Reply:
Right! I overlooked that. Definitely makes sense! Good to merge to me!